controller/src/controller/services/lab.py

```diff
@@ -637,12 +666,19 @@ async def reconcile(self) -> None:
         for username in sorted(to_monitor):
             await self._monitor_pending_spawn(username)
 
-        # Finally, for all labs in failed or terminated state (spawn failed,
-        # killed by the idle culler, killed by the OOM killer, etc.), clean up
-        # the lab as long as the user hasn't started some other operation in
-        # the meantime.
+        # For all labs in failed or terminated state (spawn failed, killed by
+        # the idle culler, killed by the OOM killer, etc.), clean up the lab
+        # as long as the user hasn't started some other operation in the
+        # meantime.
         await self._delete_completed_labs()
 
+        # Finally, count the active labs and log a metric.
+        count = 0
+        for state in self._labs.values():
+            if state.state and state.state.status == LabStatus.RUNNING:
+                count += 1
+        await self._events.active.publish(ActiveLabsEvent(count=count))
```

Comment:
Would we want counts for labs with other statuses? If so, what about adding a `status` field to the event, and folks can filter on whatever they want?

Reply:
Why wouldn't we log separate metrics for each type of lab? Is there some benefit to combining them that I'm missing?

Reply:
Similar to the success/failure case, if it never or rarely makes sense to combine them, then I'm sold on separate events for counts of different statuses.